db/bigtable_eth1.go

```diff
@@ -3491,6 +3498,23 @@ func (bigtable *Bigtable) GetMethodLabel(id []byte, invokesContract bool) string
 	return method
 }
 
+// get a event label for its byte signature with defaults
+func (bigtable *Bigtable) GetEventLabel(id []byte) string {
+	label := ""
+	if len(id) > 0 {
+		method := fmt.Sprintf("0x%x", id)
+		cacheKey := fmt.Sprintf("EVENT:HASH_TO_LABEL:%s", method)
+		if _, err := cache.TieredCache.GetWithLocalTimeout(cacheKey, time.Hour, &method); err != nil {
+			sig, err := bigtable.GetSignature(method, types.EventSignature)
+			if err == nil && sig != nil {
+				label = *sig
+			}
```

Comment:
And that is ok, because we will not get labels for all events/methods ... and then "" (not nil ) would be written in the cache, which is also ok because we don't want to try for another hour as this is nothing that changes that frequently.